src/entity/application_paranoia.ts

```diff
@@ -7,6 +7,7 @@ import { ApplicationEntity } from './application_entity';
  */
 @Entity()
 export abstract class ApplicationParanoia extends ApplicationEntity {
-  @Column({ type: 'timestamptz', default: null })
+  // @Column({ type: 'timestamptz', default: null })
+  @Column({ default: null })
```

Comment:
```suggestion
  @Column({ type: 'timestamptz', default: null })
```

chỗ này anh để type timestamptz để xóa mềm